packages/dappmanager/src/modules/compose/labelsDb.ts

```diff
@@ -48,7 +52,11 @@ const labelParseFns: {
   "dappnode.dnp.avatar": parseString,
   "dappnode.dnp.origin": parseString,
   "dappnode.dnp.chain": value =>
-    value && chainDrivers.includes(value as ChainDriver)
+    value &&
+    (chainDriversTypes.includes(value as ChainDriverType) ||
+      chainDriversTypes.includes(
+        (parseJsonSafe(value) as ChainDriverSpecs).driver as ChainDriverType
+      ))
```

Comment:
Not sure this code works. Please add unit tests for both cases where chainDriver is a string and an object

Reply:
fixed and added unit tests